.env.oidc.default

```diff
@@ -0,0 +1,5 @@
+OAUTH2_CLIENT_PROVIDER_OIDC_ISSUER=https://login.microsoftonline.com/524e761c-d162-4fdf-ab43-2855246d986c/v2.0
+OAUTH2_CLIENT_REGISTRATION_LOGIN_CLIENT_ID=505041fc-fca2-4a74-88ee-6d50a6417e38
+OAUTH2_CLIENT_REGISTRATION_LOGIN_CLIENT_SECRET=3t~CH3Tkz-Fx7CVA9VjXjCW_CK_8kS8fD.
```

Comment:
Presumably this is for our Playground env?

Reply:
At the moment not used at all - I am re-using the old AAD variables. This is a leftover and needs to be cleaned up.

Reply:
https://app.zenhub.com/workspaces/cherrytwist-5ecb98b262ebd9f4aec4194c/issues/cherrytwist/server/765